docs/language/accounts.mdx

```diff
@@ -167,6 +167,16 @@ struct AccountKey {
 struct PublicKey {
     let publicKey: [UInt8]
     let signatureAlgorithm: SignatureAlgorithm
+    let isValid: Bool
+
+    /// Verifies a signature. Checks whether the signature was produced by signing
+    /// the given tag and data, using this public key and the given hash algorithm
+    pub fun verify(
```

Comment:
As @turbolent mentioned about breaking changes, I think the change from `isValid` to `verify` is a breaking one.

Reply:
Hmm, both were added, so this isn't a breaking change

Reply:
ah my bad, I somehow though `isValid` already existed.